rosbag2_tests/CMakeLists.txt

```diff
@@ -42,7 +42,8 @@ if(BUILD_TESTING)
 
   ament_add_gmock(test_rosbag2_record_end_to_end
     test/rosbag2_tests/test_rosbag2_record_end_to_end.cpp
-    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    TIMEOUT 120)
```

Comment:
Note: new test suite just takes longer than the default minute, given the doubled number of tests

Reply:
It looks like this timeout needs to be increased further for this test to pass.

Reply:
I was wrong - there was a broken test which caused this suite to time out. I've pushed a commit to fix it.